### DISCUSSION TITLE
ignore GIT_AUTHOR_NAME and use local config instead

### DISCUSSION THREAD

**Discussion by fauzanashari** [coder/code-server#7075] [Q&A]
Hello, I just working in coder when I set git username with `git config --local user.name fauzan` it's changed then I doing my several commit. Something wrong here when I see the logs the author name is not same as I configured before. After debugging with `env | grep GIT` I just found that variable `GIT_AUTHOR_NAME` whom change it's author name in logs. The question is how to ignore or unset `GIT_AUTHOR_NAME` and use git local config instead? thank you anyway

**Comment by code-asher** [comment]
Is this for Coder or code-server?  Either way, I do not believe we set `GIT_AUTHOR_NAME`, but if you mean Coder then it could be that your workspace template has it defined or it might be using the [git-config](https://github.com/coder/modules/tree/main/git-config) module.

So you could modify the template, but if you are not able to do that then you might have to override it in your `~/.bashrc`, maybe.

**Reply by code-asher** [reply]
You might also be able to configure it as a parameter, depends on your template setup though.
